src/actions/guides/http_and_routing/http_handlers.cr

```diff
@@ -36,19 +36,19 @@ class Guides::HttpAndRouting::HTTPHandlers < GuideAction
     ```
 
     The `request` object will start in the `Lucky::ForceSSLHandler`, do some processing, then move on to the `Lucky::LogHandler`, and so on.
-    In the `Lucky::RouteHandler` it will look for a `Lucky::Action` that mathes the request, run any pipes, then run the action. If no action is found,
+    In the `Lucky::RouteHandler` it will look for a `Lucky::Action` that matches the request, run any pipes, then run the action. If no action is found,
     we check to see if there's a static file in `Lucky::StaticFileHandler`. Finally, if no route or file matches the request, we run the `Lucky::RouteNotFoundHandler`.
 
     ## Creating custom handlers
 
-    Your application may have special requirements like routing legacy URLs, sending bug reporting, CORS, or even doing HTTP Basic auth while your app is in beta.
-    Whatever your usecase, creating a custom handler is really easy!
+    Your application may have special requirements like routing legacy URLs, sending bug reporting, CORS, or even doing basic HTTP auth while your app is in beta.
```

Comment:
This should  remain and maybe link to https://en.wikipedia.org/wiki/Basic_access_authentication so it is clear that it is talking about a specific  kind  of authentication over HTTP